### DISCUSSION TITLE
Customize columns on Automation page - and others

### DISCUSSION THREAD

**Discussion by msavazzi** [home-assistant/frontend#13857] [Configuration]
I would like to have the description shown in the Automations page if possible.
In terms of a more generic features it would be great to be able to choose which colums can be shown in the configuration pages like Devices, Entities, Automation, Scripts, etc…

I hope is not too complex to do

M

**Comment by ildar170975** [comment]
I would be happy to see a "Create date" for entities & automation.
But I do not know if this info is available in DB.
